lib/schemaUtils.js

```diff
@@ -2108,7 +2122,27 @@ module.exports = {
     });
 
     responseBodyWrapper = this.convertToPmResponseBody(response.content, components, options, schemaCache);
+    previewLanguage = this.resolveResponsePreviewLanguageAndResponseHeader(responseBodyWrapper,
+      responseHeaders, response);
+    // replace 'X' char with '0'
+    code = code.replace(/X/g, '0');
```

Comment:
An improvement: we can also replace `x` (small case) with `0` here.

Reply:
Added